index.html

```diff
@@ -237,36 +223,36 @@ <h3>Introduction</h3>
 <p>The <a>PerformanceResourceTiming</a> interface facilitates
 timing measurement of downloadable resources. For example, this
 interface is available for <a data-cite=
-"!XHR/#interface-xmlhttprequest">XMLHttpRequest</a> objects
-[[XHR]], HTML elements [[HTML]] such as <a data-cite=
-"!HTML/#the-iframe-element">iframe</a>, <a data-cite=
-"!HTML/#the-img-element">img</a>, <a data-cite=
-"!HTML/#the-script-element">script</a>, <a data-cite=
-"!HTML/#the-object-element">object</a>, <a data-cite=
-"!HTML/#the-embed-element">embed</a>, and <a data-cite=
-"!HTML/#the-link-element">link</a> with the link type of
-<a data-cite="!HTML/#link-type-stylesheet">stylesheet</a>, and SVG
+"XHR#interface-xmlhttprequest">XMLHttpRequest</a> objects [[XHR]],
+HTML elements [[HTML]] such as <a data-cite=
+"HTML#the-iframe-element">iframe</a>, <a data-cite=
+"HTML#the-img-element">img</a>, <a data-cite=
+"HTML#the-script-element">script</a>, <a data-cite=
+"HTML#the-object-element">object</a>, <a data-cite=
+"HTML#the-embed-element">embed</a>, and <a data-cite=
+"HTML#the-link-element">link</a> with the link type of
+<a data-cite="HTML#link-type-stylesheet">stylesheet</a>, and SVG
 elements [[SVG11]] such as <a data-cite=
-"!SVG11/struct.html#SVGElement">svg</a>.</p>
+"SVG11/struct.html#SVGElement">svg</a>.</p>
 </section>
 <section>
 <h3>Resources Included in the <a>PerformanceResourceTiming</a>
 Interface</h3>
-<p>All resources <a data-cite="!FETCH/#concept-fetch">fetched</a>
-by the current <a data-cite="!HTML/#browsing-context">browsing</a>
-or worker [[!WORKERS]] context's MUST be included as
+<p>All resources <a data-cite="FETCH#concept-fetch">fetched</a> by
+the current <a data-cite="HTML#browsing-context">browsing</a> or
+worker [[WORKERS]] context's MUST be included as
 <a>PerformanceResourceTiming</a> objects in the <a data-cite=
-"!PERFORMANCE-TIMELINE-2/#performance-timeline">Performance
+"PERFORMANCE-TIMELINE-2#performance-timeline">Performance
 Timeline</a> of the relevant context. Resources that are retrieved
-from <a data-cite="!HTML/#relevant-application-cache">relevant
+from <a data-cite="HTML#relevant-application-cache">relevant
 application caches</a> or local resources MUST be included as
 <a>PerformanceResourceTiming</a> objects in the <a data-cite=
-"!PERFORMANCE-TIMELINE-2/#performance-timeline">Performance
-Timeline</a> [[!PERFORMANCE-TIMELINE-2]]. Resources for which the
-<a data-cite="!FETCH/#concept-fetch">fetch</a> was initiated, but
-was later aborted (e.g. due to a network error) MAY be included as
+"PERFORMANCE-TIMELINE-2#performance-timeline">Performance
```

Comment:
While we're at it, is it possible to point to the WD, rather than the TR? (Potentially at a different PR)

Reply:
To be clear, I can do that, just asking if it's possible (and if so, what's the magic spec name)

Reply:
Yes, but you need to check the key on https://www.specref.org and add it if missing.

Reply:
Looking at specref, it seems like the ED URL is already there in the w3c.json, but as "edDraft" rather then the "href". Is it possible to point to that URL from the caller? Or do I need to add a separate entry where the "href" is the ED draft URL?